src/features/cookies/helpers/useCookies.ts

```diff
@@ -17,6 +18,14 @@ const COOKIES_CONSENT_KEY = 'cookies'
 export const getCookiesChoice = async () =>
   await storage.readObject<CookiesConsent>(COOKIES_CONSENT_KEY)
 
+const removeCookiesConsent = (cookiesChoice: CookiesConsent) => {
+  storage.saveObject(COOKIES_CONSENT_KEY, {
```

Comment:
il y a une promesse qui est déclenchée mais pas attendue, ça risque de faire des effets de bords indésirables

je recommande d'attendre que le save soit fini

```suggestion
const removeCookiesConsent = async (cookiesChoice: CookiesConsent): Promise<void> => {
  await storage.saveObject(COOKIES_CONSENT_KEY, {
```